[2024-06-14T19:56:12.499+0530] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-14T19:56:12.557+0530] {taskinstance.py:2063} INFO - Dependencies not met for <TaskInstance: my_data_pipeline_dag.collect_data_3 manual__2024-06-14T14:25:30.143337+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-06-14T19:56:12.559+0530] {local_task_job_runner.py:163} INFO - Task is not able to be run
